### PR TITLE
Update ww_head.py

### PR DESCRIPTION
**Pull request by ryandezinator**

### CODE REVIEW

File: interpretation/ww_head.py [hunk 1]
```diff
@@ -860,7 +860,7 @@ def process(message, isGameMaster=False, isAdmin=False):
         # TODO
         return todo()
     if is_command(message, ['age'], True):
-        msg = "**USAGE:** This command is used to set your age. /n/n`" + prefix + "age<number>\`n\n**Example:** `!age 19`"
```
Comment (Bentechy66) [inline interpretation/ww_head.py]: Can you add replace this line again please?

File: interpretation/ww_head.py [hunk 2]
```diff
@@ -871,8 +871,9 @@ def process(message, isGameMaster=False, isAdmin=False):
         # TODO
         return todo()
     if is_command(message, ['profile'], True):
-        # TODO
-        return todo()
+        msg = "**USAGE:** The use of this command is so that you can view your own profile, if you add another players name, you can view their profile . /n/n` + prefix + /n/n **Example:** `!profile or !profile Randium"
```
Comment (Bentechy66) [inline interpretation/ww_head.py]: The `/n` should be `\n`. Also there is a space after the word "Profile"

File: interpretation/ww_head.py (hunk 1 above)
Comment (ryandezinator) [reply interpretation/ww_head.py]: sure

Comment (Randium) [reply interpretation/ww_head.py]: This one still has /n instead of \n

File: interpretation/ww_head.py (hunk 2 above)
Comment (Randium) [reply interpretation/ww_head.py]: This one still has /n instead of \n

Comment (ryandezinator) [reply interpretation/ww_head.py]: again, that has been fixed in the lates pull request!

File: interpretation/ww_head.py (hunk 1 above)
Comment (Randium) [reply interpretation/ww_head.py]: Nevermind. It doesn't, but I don't see a replaced message?

File: interpretation/ww_head.py (hunk 2 above)
Comment (Randium) [reply interpretation/ww_head.py]: You mean that you created a different pull request? That's kinda messy. Why didn't you alter this pull request?

**Comment by Randium** [comment]
This pull request is closed due to a different pull request fixing said issues.

File: interpretation/ww_head.py (hunk 2 above)
Comment (ryandezinator) [reply interpretation/ww_head.py]: cause i made that pull request before knowing this problem existed, but before even knowing this was a problem, i changed it cause i noticed it was different to the rest

**Pull request closed without merging**